### DISCUSSION TITLE
Loving the orphan fragments feature! Could it be mentioned briefly in `towncrier create --help`?

### DISCUSSION THREAD

**Discussion by blakeNaccarato** [twisted/towncrier#593] [Ideas]
I appreciate how concise the CLI help is currently, but maybe orphan fragments are worth mentioning? I see how this requires another level of detail in the help text, since mentioning orphan fragments means you also should mention non-orphaned fragments as well. The detailed summary could have added clarification along the lines of:

```Text
For example, a FILENAME of "123.feature.md" will link to #123 in your issue tracker,
while a "+"-prefixed FILENAME like "+.feature.md" will create an "orphan fragment",
and it won't link to any particular item.
```

So an alternative view is that the brevity of the current help text is intentional and should be maintained. For reference, here is the orphaned fragment PR and related mentions of orphaned fragments in docs so far:

- https://github.com/twisted/towncrier/pull/428
- [Creating news fragments](https://towncrier.readthedocs.io/en/stable/tutorial.html#creating-news-fragments)
- [`towncrier` 22.12.0rc1 (2022-12-20) features](https://towncrier.readthedocs.io/en/stable/release-notes.html#id48)

**Comment by adiroiban** [comment]
I think it is mentioned ... but in the dev version

https://github.com/twisted/towncrier/blob/db10a4d872450eac0441a8f873c2e0c0eba78889/src/towncrier/create.py#L75-L76

-------

I guess it's time for a new release :)

**Reply by blakeNaccarato** [reply]
Ah, I missed that, thanks for clarifying!
